downstream/titles/release-notes/topics/errata-24-aug-28.adoc

```diff
@@ -0,0 +1,105 @@
+// This is the release notes file for AAP 2.4 errata bundle dated August 28 2023.
+
+= {PlatformNameShort} 2.4 - August 28, 2023
+
+link:https://access.redhat.com/errata/RHBA-2023:4782[Red Hat Errata Release]
+
+== {ControllerNameStart}
+
+.New features and enhancements
+
+* automation-controller has been updated to 4.4.2.
+
+.Security fixes
+
+* automation-controller: python-django: Potential regular expression denial of service vulnerability in EmailValidator/URLValidator (CVE-2023-36053)
+
+* automation-controller: python-django: Potential denial-of-service vulnerability in file uploads (CVE-2023-24580)
+
+.Bug fixes
+
+* Changing credential types by using the drop-down list in the *Launch prompt* window no longer causes the screen to disappear.
+
+* Upgraded python dependencies which include upgrades from Django 3.2 to 4.2.3, psycopg2 to psycopg3, and additional libraries as needed. Also added a new setting in the UI exposing the `CSRF_TRUSTED_ORIGIN` settings.
+
+* Fixed slow database UPDATE statements on the job events table which could cause a task manager timeout.
+
+* Fixed an issue where adding a new label to a job through the *Prompt On Launch* option would not add the label to the job details.
```

Comment:
This is an awesome resolution to the confusion of this line before! Super job.

Reply:
Thanks so much for saying so :D